packages/plugin-graphql/test/cases/query-config/query-config.spec.js

```diff
@@ -82,7 +82,7 @@ describe('Build Greenwood With: ', function() {
       });
 
       it('should have one window.__APOLLO_STATE__ <script> with (approximated) expected state', function() {
-        const scriptTags = dom.window.document.querySelectorAll('script');
+        const scriptTags = dom.window.document.querySelectorAll('head script');
```

Comment:
This looks good!  If you look in the rest of the test cases for this package, I think you will find a few more test cases with this exact same code, so if you do a search for something file _"should have one window.__APOLLO_STATE__ <script>"_ you should easily be able to find them and the adjust those too.  👍

Reply:
Hey, I'm not being able to find more such files where I can make changes can you please help out, so I can make change

Reply:
Yeah, If you look in the other [test case files in the **plugin-graphql** package](https://github.com/ProjectEvergreen/greenwood/tree/master/packages/plugin-graphql/test/cases) you will find other instances of the kind of test you see
- https://github.com/ProjectEvergreen/greenwood/blob/master/packages/plugin-graphql/test/cases/query-children/query-children.spec.js#L155
- https://github.com/ProjectEvergreen/greenwood/blob/master/packages/plugin-graphql/test/cases/query-custom-frontmatter/query-custom-frontmatter.spec.js#L157
- https://github.com/ProjectEvergreen/greenwood/blob/master/packages/plugin-graphql/test/cases/query-custom-schema/query-custom-schema.spec.js#L75
- https://github.com/ProjectEvergreen/greenwood/blob/master/packages/plugin-graphql/test/cases/query-graph/query-graph.spec.js#L153
- https://github.com/ProjectEvergreen/greenwood/blob/master/packages/plugin-graphql/test/cases/query-menu/query-menu.spec.js#L155